design-reviews/recipes/engagement-process.md

```diff
@@ -0,0 +1,129 @@
+1. Present any new design artifacts including API contracts, architecture diagrams, workflows, deployment pipelines, etc.
+1. Describe if there are any high level architecture or design changes since last review
+1. How have your design changes impacted the initial game plan
+
+## Sprint Planning
+
+In many of our engagements we will be working with our customers using a SCRUM agile development process.  Sprint planning is a great opportunity to dive deep into the next set of high priority work.  Some key points to address are the following:
+
+1. Identify stories that require design reviews
+1. Breakout design from implementation for complex stories
+1. Assign owners to each design story
+
+Stories that will benefit from design reviews have one or more of the following in common:
+
+1. Are there many unknowns or unclear requirements
+1. There is wide distribution in story point costing amongst your dev crew
+1. You cannot clearly illustrate all the tasks required for a story
+
+> **Tip:** After your sprint planning is complete your team should consider hosting an initial design review discussion to dive deep on the design requirement of the stories that were identified.  After you have more clarity the team should continue forward with either a sync or async design review to complete the tasks.
+
+## Sprint Backlog Grooming
+
+If you team is not already hosting a Sprint Backlog Grooming session at least once per week you should consider it.  It is a great opportunity to:
+
+1. Keep your backlog clean
+1. Re-prioritize work bash on shifting business priorities
+1. Fill in missing descriptions / acceptance criteria
+1. Identify stories that require design reviews
+
+The team can follow the same steps from [sprint planning](#sprint-planning) to help identify which stories require design reviews.  This can often save much time during your actual sprint planning meetings to focus on the task at hand.
+
+## Sprint Retrospectives
+
+Sprint retrospectives are a great time to check-in with your dev team to identify what is working or not working within your group and propose changes to keep improving.
+
+It is also a great time to check-in on your design reviews
+
+- Did any of your designs change from last sprint?
+- How have your design changes impacted your engagement?
+- Have previous design artifacts been updated to reflect new changes?
+
+All design artifacts including the game plan should be treated as a living document.  As requirements change or you uncover more unknowns the dev crew should be going back and updating all your design artifacts to reflect reality.  If this step is missed you are making a critical mistake which could lead to the customer incurring more technical debt in the future. Artifacts that are not up to date are basically `bugs` in your design.
+
+> **Tip:** Keep your artifacts up to date by adding it to your teams `Definition of Done` for all user stories.

```

Comment:
Maybe link Definition of Done to https://github.com/microsoft/code-with-engineering-playbook/tree/master/team-agreements/definition-of-done